R/checks.R

```diff
@@ -568,6 +568,20 @@ checkarg_file_name <-
     }
   }
 
+#' Check if the temporary directory exists:
+#' @importFrom rlang abort
+#' @keywords internal
+checkarg_tempdir <-
+  function(tmp_dir){
+    if(!dir.exists(tmp_dir)){
+      rlang::abort(
+        c("Error in `tempdir`:",
```

Comment:
```suggestion
        c("Error in `tmp_dir`:",
```
Is this right? We want to use the arg name from `fetch_survey()`?

Reply:
Yes, my mistake.  The new arg was originally `tempdir`, but that had problems due to overlap with the base function.  Thought I caught all the instances after the change but I missed this one.